sky/packages/sky/example/game/main.dart

```diff
@@ -93,27 +94,37 @@ class GameDemoApp extends App {
     return new Stack([
       new SpriteWidget(new MainScreenBackground()),
       new Center(
-        child: new TextureButton(
-          onPressed: () {
-            _game = new GameDemoWorld(
-              _app,
-              navigator,
-              _loader,
-              _spriteSheet,
-              _spriteSheetUI
-            );
-            navigator.pushNamed('/game');
-          },
-          texture: _spriteSheetUI["btn_play_up.png"],
-          textureDown: _spriteSheetUI["btn_play_down.png"],
-          width: 128.0,
-          height: 128.0
-        )
+        child: new Flex([
+          new TextureButton(
+            onPressed: () {
+              _game = new GameDemoWorld(
+                _app,
+                navigator,
+                _loader,
+                _spriteSheet,
+                _spriteSheetUI
+              );
+              navigator.pushNamed('/game');
+            },
+            texture: _spriteSheetUI["btn_play_up.png"],
+            textureDown: _spriteSheetUI["btn_play_down.png"],
+            width: 128.0,
+            height: 128.0
+          ),
+          new Text(
+            "Last Score: $lastScore",
+            style: new TextStyle(fontSize:20.0)
+          )
+        ],
+        direction: FlexDirection.vertical,
+        justifyContent: FlexJustifyContent.center)
       )
     ]);
   }
 }
 
+int lastScore = 0;
```

Comment:
Rather than setting a global variable, the pattern we usually use is to pass in a callback that something lower down in the tree can call when a new score is available.  Then the GameDemoApp can get the callback and setState itself to set its private _lastStore state.